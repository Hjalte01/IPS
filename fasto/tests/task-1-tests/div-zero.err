Type error: Attempt to divide by 0 at line 2, column 10
