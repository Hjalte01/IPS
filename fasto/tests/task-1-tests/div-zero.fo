fun int main() =
  write(4/0)
