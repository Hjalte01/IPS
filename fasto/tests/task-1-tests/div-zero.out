Interpreter error: Attempt to divide by 0 at line 4, column 13
